src/Logger/BanditActionEvent.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Eppo\Logger;
+
+use DateTime;
+use Eppo\DTO\Bandit\Bandit;
+use Eppo\DTO\Bandit\BanditEvaluation;
+use JsonSerializable;
+use Serializable;
+
+class BanditActionEvent implements Serializable, JsonSerializable
+{
+    public function __construct(
+        public readonly string $flagKey,
+        public readonly string $banditKey,
+        public readonly string $subjectKey,
+        public readonly ?string $action,
+        public readonly ?float $actionProbability,
+        public readonly ?float $optimalityGap,
+        public readonly ?string $modelVersion,
```

Comment:
Now that we are not logging non-bandit actions, these will always be defined and don't need to be optional